pages/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Redirecting...</title>
+  <link rel="canonical" href="./main/index.html">
```

Comment:
Is it standard to have the `.` here?
As far as I remember you'd use `main/index.html` for relative paths and `/main/index.html` for absolute paths